.dockerignore

```diff
@@ -0,0 +1,5 @@
+# Git
+.git/
+
+# Node
+frontend/node_modules
```

Comment:
This should be put into the `.gitignore` file which is found in the `flotilla` folder. https://github.com/oysand/flotilla/blob/dockerize-frontend/frontend/.gitignore

Reply:
Or this was a `.dockerignore`... 

Should the `.dockerignore` be together with the compose file in the root or together with the dockerfile in frontend?

Reply:
This serves two different purposes. This is the .dockerignore

Reply:
Just saw the second comment.. I think it makes sense to have it together with the compose file as there will be a second docker container for the backend, but the compose file ties them both together and we can use the same docker ignore for both